9c-internal/9c-single/values.yaml

```diff
@@ -3,74 +3,204 @@
 # Declare variables to be passed into your templates.
 
 
-clusterName: "9c-internal-v2"
-logLevel: ""
-namespace: "9c-single"
+clusterName: "9c-sample"
```

Comment:
Is this cluster name correct?

Reply:
oh, thanks, let me update it. but it doesnt matter as it works XD